networkx/linalg/laplacianmatrix.py

```diff
@@ -1,4 +1,8 @@
 """Laplacian matrix of graphs.
+
+All calculations here are done using the out-degree. For Laplacians using in-degree, us `G.reverse(copy=False)` instead of `G`.
+
+The `laplacian_matrix` function provides an unnormalized matrix, while `normalized_laplacian_matrix`, `directed_laplacian_matrix`, and `directed_combinatorial_laplacian_matrix` are all normalized.
```

Comment:
Did our linting rules change? These lines are clearly greater than 100 chars so I would've expected CI to complain about it. @MridulS any ideas?

Reply:
Should I go ahead and just fix the line lengths manually now?

Reply:
I don't think our lint checks line lengths for comments. But I could be wrong.

Yes, @smokestacklightnin it would help to wrap the comment lines to about 80 chars (less than 88 for sure). There are others below...

Reply:
Done